src/test/java/de/zalando/ep/zalenium/proxy/DockerSeleniumStarterRemoteProxyTest.java

```diff
@@ -1,459 +1,414 @@
 package de.zalando.ep.zalenium.proxy;
```

Comment:
@dspasojevic Any reason why this isn't deleted altogther yet?

Reply:
Was waiting to see if there was something that could be salvaged from here. I'll look at replacing with higher value integration tests.